front/lib/api/assistant/configuration.ts

```diff
@@ -542,44 +477,10 @@ async function fetchWorkspaceAgentConfigurationsForView(
     const actions: AgentActionConfigurationType[] = [];
 
     if (variant === "full") {
-      const retrievalConfigurations = retrievalConfigs[agent.id] ?? [];
-      for (const retrievalConfig of retrievalConfigurations) {
-        const dataSourcesConfig =
-          retrievalDatasourceConfigurations[retrievalConfig.id] ?? [];
-        let topK: number | "auto" = "auto";
-        if (retrievalConfig.topKMode === "custom") {
-          if (!retrievalConfig.topK) {
-            // unreachable
-            throw new Error(
-              `Couldn't find topK for retrieval configuration ${retrievalConfig.id}} with 'custom' topK mode`
-            );
-          }
+      const retrievalActions =
```

Comment:
these are not actions, they are action configurations (naming is quite confusing IMO)